### ISSUE TITLE
About train_speaker_embeddings train_ecapa_tdnn_big.yaml data enhancement file 

### ISSUE THREAD

**Issue by guowhite** [speechbrain/speechbrain#750]
in readme.md have a train_ecapa_tdnn_big.yaml file which is concatenate waveform dropout, speed change, reverberation, noise, and noise+rev. but i can't find it in hparams path

**Comment by mravanelli** [comment]
Hi,
the augmentation is specified here:
https://github.com/speechbrain/speechbrain/blob/develop/recipes/VoxCeleb/SpeakerRec/hparams/train_ecapa_tdnn.yaml#L78
Are you using the latest version of SpeechBrain?

**Comment by mravanelli** [comment]
The point is that I don't see train_speaker_embeddings
train_ecapa_tdnn_big.yaml in the latest version of SpeechBrain:
https://github.com/speechbrain/speechbrain/tree/develop/recipes/VoxCeleb/SpeakerRec/hparams
So the file that you are using looks outdated.

**Comment by Alva-2020** [comment]
> sorry. I see this command in readme. i am also not see train_ecapa_tdnn_big.yaml. so i just want to know how use the augmentation. like python train_speaker_embeddings.py hparams/train_ecapa_tdnn.yaml and set sample_rate is ok?

Online-data augmentation has embeded in the training step. Loading  hparams/train_ecapa_tdnn.yaml  can initialize the augmentation class. In python train_speaker_embeddings.py, compute_forward called the initialized augmentation class.

**Comment by guowhite** [comment]
thank you. i will tried use my dataset for try train model for model training
